app/models/donation.rb

```diff
@@ -9,4 +9,8 @@ class Donation < ActiveRecord::Base
   validates :amount, presence: true, numericality: { greater_than: 0 }
 
   scope :search, -> (keyword) { joins(:donor).where("name ILIKE ?", "%#{keyword}%") }
+
+  def self.donations_for_current_year
+    Donation.where("extract(year  from created_at) = ?", Time.zone.today.year).sum(:amount).round
```

Comment:
1. We're already in the `Donation` class, so you don't need to qualify it here.
2. As a convention, use uppercase for SQL keywords: `EXTRACT(YEAR FROM created_at)`
3. `Time.zone.today.year` -> `Date.current.year`